src/table/__tests__/resizable-columns.test.tsx

```diff
@@ -431,3 +437,11 @@ describe('column header content', () => {
     expect(getResizeHandle(1)).toHaveAttribute('aria-roledescription', 'resize button');
   });
 });
+
+test('should set last column width to "auto" when container width exceeds total column width', () => {
+  jest.mocked(useResizeObserver).mockImplementation((_target, cb) => cb({ contentBoxWidth: 451 } as any));
```

Comment:
If there is a non round value, it needs a comment explaining why

Reply:
Makes sense , will add